QUEST.tsv

```diff
@@ -2765,252 +2765,252 @@ QUEST_20151016_002772	$When the Royal Army said that they will spare our lives w
-QUEST_20151102_002937
-QUEST_20151102_002938
-QUEST_20151102_002939
-QUEST_20151102_002940
-QUEST_20151102_002941
-QUEST_20151102_002942
-QUEST_20151102_002943
-QUEST_20151102_002944
-QUEST_20151102_002945
-QUEST_20151102_002946
-QUEST_20151102_002947
-QUEST_20151102_002948
-QUEST_20151102_002949
-QUEST_20151102_002950
-QUEST_20151102_002951
-QUEST_20151102_002952
-QUEST_20151102_002953
-QUEST_20151102_002954
-QUEST_20151102_002955
-QUEST_20151102_002956
-QUEST_20151102_002957
-QUEST_20151102_002958
-QUEST_20151102_002959
-QUEST_20151102_002960
-QUEST_20151102_002961
-QUEST_20151102_002962
-QUEST_20151102_002963
-QUEST_20151102_002964
-QUEST_20151102_002965
-QUEST_20151102_002966
-QUEST_20151102_002967
-QUEST_20151102_002968
-QUEST_20151102_002969
-QUEST_20151102_002970
-QUEST_20151102_002971
+QUEST_20151102_002916	First of all, magic must be strong.{nl}Using powerful spells to unleash powerful magic is basic.{nl}What can you do with weak magic?{nl}
+QUEST_20151102_002917	To become a stronger mage, you cannot idle around and ignore practice.{nl}Well, you might still be able to catch up to talented people such as myself.
+QUEST_20151102_002918	It's very important to maintain a relaxed mind when using magic.{nl}Not only can you not be able to use magic when losing your composure, you might also find yourself in danger.{nl}Those who use magic should keep their heads and hearts calm at all times.
+QUEST_20151102_002919	Delicate control is required for higher ranked magic.{nl}A minor mistake may lead to a fatal result.{nl}You must not forget this as you get more skillful and handle more variety of magic.
+QUEST_20151102_002920	An advice from me?{nl}Speaking as a Linker, you must have a wide perspective.{nl}There are many cases of those mages who uses synergy magic, without checking their surrounding as they want to cast magic as soon as possible.{nl}
+QUEST_20151102_002921	Not only do you have to watch for yourself and your enemies, but you must as well have to be in the head of your comrade in arms.{nl}If you don't plan the effects of your magic ahead, you are no different than those reckless novice soldiers swinging their swords around carelessly.
+QUEST_20151102_002922	$Psychokino Tome Volume 1
+QUEST_20151102_002923	$Story of the first Psychokino
+QUEST_20151102_002924	$Psychokino Tome Volume 2
+QUEST_20151102_002925	$Story of Psychokino's magic
+QUEST_20151102_002926	$Psychokino Tome Volume 3
+QUEST_20151102_002927	$Story of Psychokinos before the Medzio Diena
+QUEST_20151102_002928	$Psychokino Tome Volume 4
+QUEST_20151102_002929	$What Psychokino did at the Mage Tower
+QUEST_20151102_002930	$Psychokino Tome Volume 5
+QUEST_20151102_002931	$Psychokino's Potential
+QUEST_20151102_002932	After the lord's migration has fallen, the people of the south are heading towards the Orsha.{nl}If you're an adventurer who has recently started your travel, head to camping area of the east.{nl}There should be people who's able to guide you.
```

Comment:
the Orsha -> Orsha
your travel -> traveling

of the east -> on the east?
There should be people who's able to guide you. -> There should be people who are able to guide you.